rust-toolchain.toml

```diff
@@ -1,3 +0,0 @@
-[toolchain]
```

Comment:
This is a problem that needs resolving. It seems `cargo fuzz` requires running on nightly, but our rust-toolchain forces you to run against `1.57` - which stops the fuzz test from runnning.

Would welcome solutions to this!

Reply:
🤔 Is it not possible to just pass `+nightly` when running `cargo` to build the fuzz tests? We could add this as a `make` target.

Reply:
Sweet! Yes that works!